about.html

```diff
@@ -12,57 +12,80 @@
 </head>
 
 <body>
-    <div class="container about-page">
+   <div class="container2">
+      <header class="header">
+          <img src="/img/lambda-black.png">
+          <nav class="nav">
+              <a class="navie" style="text-decoration: none;" href="/index.html">Home</a>
```

Comment:
If you have a CSS file already, you don't need the inline style. You can just select it in your CSS and do the same thing. Great job on that inline style syntax though!